.github/workflows/push.yaml

```diff
@@ -0,0 +1,128 @@
+name: push-pipeline
+on:
+  push:
+    branches:
+      - master
```

Comment:
Here also, we should have `main` branch instead of `master`?

Reply:
Yes, changed.